index.js

```diff
@@ -0,0 +1,17 @@
+'use strict'; // eslint-disable-line strict
+
+const werelogs = require('werelogs');
+
+const runServer = require('./lib/api/BackbeatServer');
```

Comment:
I think it would be worthwhile to put the run function in another file because a file name like `./lib/api/BackbeatServer` seems like it is exporting the class `BackbeatServer`.

Reply:
I agree, i'd like to implement this in a refactor after everything is merged though. There is some refactoring, renaming, and reorganizing to be done :S